ci/azure/windows_mingw_script

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+set -x
+set -e
+
+pacman --noconfirm --needed -S git base-devel mingw-w64-x86_64-toolchain mingw-w64-x86_64-cmake mingw-w64-x86_64-clang mingw-w64-x86_64-lld mingw-w64-x86_64-llvm
+
+git config core.abbrev 9
+
+ZIGBUILDDIR="$(pwd)/build"
+PREFIX="$ZIGBUILDDIR/dist"
+CMAKEFLAGS="-DCMAKE_COLOR_MAKEFILE=OFF -DCMAKE_INSTALL_PREFIX=$PREFIX -DZIG_STATIC=ON"
+
+mkdir $ZIGBUILDDIR
+cd $ZIGBUILDDIR
+
+if [ "${BUILD_REASON}" == "PullRequest" ]; then
```

Comment:
why is this logic different depending on whether it's a pull request?

Reply:
i think blindly copied that from one of the other CI scripts, but it shouldn't be needed now because these executables aren't going to be packaged.